src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -484,6 +480,45 @@ def _restart_qemu(self):
     start_qemu()
     self._setup_device_and_fuzzer()
 
+  def _corpus_target_subdir(self, relpath):
+    return os.path.join(self.corpus_directories_target, relpath)
+
+  def _init_corpus_info(self, corpus_directories):
+    """ Initialize corpus info for fuzzing and merging. """
+    self.corpus_directories = corpus_directories
+    # By convention, the *first* corpus directory in corpus_directories is
+    # always where new corpus files from the target are stored.
+    self.new_corpus_dir_host = corpus_directories[0]
```

Comment:
similarly for these other variables, just calculate them as needed.

Reply:
got it, made individual functions to calculate these vars